### PR TITLE
Wishlist + Cart Branch

### PR DESCRIPTION
**Pull request by tejas-muthya37**
This branch has the screens for Wishlist and Cart pages for my E-Commerce App.

### CODE REVIEW

**Review by Hnh013 — APPROVED** [review]
looks fine by me , good to merge

File: Cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Biblio | Books Delivered</title>
+    <!-- <link rel="stylesheet" href="https://magic-library.netlify.app/master.css"> -->
```
Comment (romabulani) [inline Cart/cart.html]: You can remove the comments if not required

File: Navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        position: absolute;
+        right: 0px;
+        top: 9vh;
+        height: 91vh;
+        background-color: var(--body-bg);
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: space-around;
+        width: 300px;
```
Comment (romabulani) [inline Navbar/navbar.css]: If possible, use rems here

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge

File: Navbar/navbar.css (hunk 2 above)
Comment (tejas-muthya37) [reply Navbar/navbar.css]: Done. Thanks.

**Pull request merged**